pkgs/by-name/ma/managarr/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  nix-update-script,
+  perl,
+}:
+
+let
+  pname = "managarr";
+  version = "0.4.2";
+in
+
+rustPlatform.buildRustPackage {
+  inherit pname version;
+  src = fetchFromGitHub {
+    owner = "Dark-Alex-17";
+    repo = "managarr";
+    rev = "v${version}";
```

Comment:
```suggestion
    tag = "v${version}";
```

Reply:
I left this because of this https://github.com/Mic92/nix-update/issues/304 which this package uses, but can change it.